app/generator/content/part2/get-table/map-total-row.js

```diff
@@ -0,0 +1,14 @@
+const mapTotalRow = (row) => {
```

Comment:
needs to handle if `row` is `undefined` -- used in `getTable` via  a `[].find` which returns `undefined` if it doesn't find anything that satisfies